src/hal/stm32f7/init.rs

```diff
@@ -0,0 +1,348 @@
+// Zinc, the bare metal stack for rust.
+// Copyright 2014 Vladimir "farcaller" Pouzanov <farcaller@gmail.com>
```

Comment:
I copied the stm32f4 file and made a few tweaks, so I can hardly claim authorship.